Examples/Recursion.java

```diff
@@ -0,0 +1,20 @@
+class HelloWorld{
+
+  public static void main(String[] args){
+  
+  static int fact(int n)
```

Comment:
Shouldn't the return type be **long** .   Otherwise, would it be able to return fatorial to numbers greater than 100?